rstudio/init.sh

```diff
@@ -0,0 +1,30 @@
+# download rstudio 
+wget http://download2.rstudio.org/rstudio-server-rhel-0.99.446-x86_64.rpm
+sudo yum install --nogpgcheck -y rstudio-server-rhel-0.99.446-x86_64.rpm
+
+# restart rstudio 
+rstudio-server restart 
+
+# add user for rstudio 
+adduser rstudio
+
+# create a Rscript that connects to Spark, to help starting user
+echo "cat('Now connecting to Spark for you.') 
+ 
+spark_link <- system('cat /root/spark-ec2/cluster-url', intern=TRUE)
+
+.libPaths(c(.libPaths(), '/root/spark/R/lib')) 
+Sys.setenv(SPARK_HOME = '/root/spark') 
+Sys.setenv(PATH = paste(Sys.getenv(c('PATH')), '/root/spark/bin', sep=':')) 
+library(SparkR) 
+
+sc <- sparkR.init(spark_link) 
+sqlContext <- sparkRSQL.init(sc) 
+
+cat('Spark Context available as \"sc\". \\n')
+cat('Spark SQL Context available as \"sqlContext\". \\n')
+"  > /home/rstudio/startSpark.R
+
```

Comment:
I think these directories are only created when Spark first runs -- could you add `mkdir -p /mnt/spark` before this ?
Also this needs to be done on `/mnt2`, `/mnt3` etc. I think -- so the right place to do this might be in `spark/setup.sh` similar to https://github.com/mesos/spark-ec2/blob/branch-1.4/ephemeral-hdfs/setup-slave.sh#L16